orca-tide/src/main/groovy/com/netflix/spinnaker/orca/tide/tasks/DeepCopyPipelineTask.groovy

```diff
@@ -36,11 +36,14 @@ class DeepCopyPipelineTask implements Task {
     Map<String, String> source = stage.context.source
     Map<String, String> target = stage.context.target
     def dryRun = stage.context.dryRun ?: false
+    def allowIngressFromClassic = stage.context.allowIngressFromClassic ?: true
```

Comment:
this is probably not right in either the case of someone trying to pass false, or someone passing the string "true" or "false"

I would make a pojo in there and use stage.mapTo